docs/apis/subsystems/tag/index.md

```diff
@@ -0,0 +1,274 @@
+---
+title: Tag API
+tags:
+- API
+- Sub System
+- Tag
+- Tags
+---
+
+## Tag API overview
+
+The Tag API allows you to assign labels to information in Moodle. This makes finding this information easier and also facilitates the grouping of similar information. The Tag API allows you to create, modify, delete and search tags in the Moodle system. The main tag related functions can be found in the tag/classes/tag.php file. For a thorough overview of all of the functions available for working with Tags please see methods in core_tag_tag, core_tag_collection and core_tag_area classes, however, the following examples should give you a general understanding of how to get started with tags.
+
+This page describes API in Moodle 3.1 and above, for earlier versions see [Tag API before 3.1](https://docs.moodle.org/dev/Tag_API_before_3.1)
```

Comment:
```suggestion
```

No need to document legacy versions of Moodle. 3.1 is 4 years old and long out of support.